packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/ThrowStatement.cs

```diff
@@ -5,8 +5,15 @@
 
 namespace Microsoft.Generator.CSharp.Statements
 {
-    public sealed record ThrowStatement(ValueExpression ThrowExpression) : MethodBodyStatement
+    public sealed class ThrowStatement : MethodBodyStatement
     {
+        public ValueExpression ThrowExpression { get; }
```

Comment:
looks like this should not be a **Throw**Expression, it should just be a `expression`. A "throw" expression might indicate this expression has a throw keyword but from the context here, this should just be the thing following keyword `throw`